src/data/real/ennreal.lean

```diff
@@ -1699,6 +1699,10 @@ lemma of_real_pow {p : ℝ} (hp : 0 ≤ p) (n : ℕ) :
   ennreal.of_real (p ^ n) = ennreal.of_real p ^ n :=
 by rw [of_real_eq_coe_nnreal hp, ← coe_pow, ← of_real_coe_nnreal, nnreal.coe_pow, nnreal.coe_mk]
 
+@[simp] lemma nsmul_of_real {x : ℝ} {n : ℕ} :
```

Comment:
I think the simp lemma is in the wrong direction: the rule is to push the operations as much to the outside as possible (just like for a linear map  `A`, we simp away `A (c • x)` to `c • A x`).

Reply:
Thanks; yes of course!

I can hardly believe I had this backwards. Now fixed.